hugo_site/content/venue.md

```diff
@@ -9,19 +9,19 @@ menu:
 
 # Venue
 
-The venue is located close to the city centre and has easy access by public transport.
+The venue is located close to the city centre and has easy access by both bicycle and public transport.
```

Comment:
Good point, as we are looking into an option of having our own special conference bike stand set up at the venue :)